eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -5,6 +5,8 @@ jobs:
     pool:
       vmImage: vs2017-win2016
     steps:
+      - checkout: self
+        path: AzureSDKEngSystem/Net/SourceDirectory
```

Comment:
What is your goal with including this?

Reply:
This is just to imitate building in a path 50 characters long in the build Agent.

Reply:
I would rather add a test to recording framework that verifies that resulting session file name length is correct. This was verification would also run on dev machines.

Reply:
I would not limit the checks to just the recording framework. While the recordings might be the biggest issue right now we need to ensure everything in the repo stays in range. @chidozieononiwu can you please make this path really long to simulate a failure in DevOps to see if the failure is diagnose-able or if it blows up the entire job in a strange way.

Reply:
I have tested that out [here](https://dev.azure.com/azure-sdk/playground/_build/results?buildId=84197&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774)